pantheon-bundle/pom.xml

```diff
@@ -287,6 +287,11 @@
             <groupId>org.apache.activemq</groupId>
             <artifactId>activemq-client</artifactId>
             <version>5.15.11</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.query</artifactId>
+            <version>3.0.0</version>
```

Comment:
I wonder if this dependency should be `provided` and added instead to the provisioning files in the `pantheon-slingstart` maven module. I think it's already an OSGI bundle.

Reply:
According to sling documentation[1], query is added with normal scope. Please correct me if I misread the document. 

Also, it seems the OSGi bundle for or a bundle that exports org.apache.sling.query is not available (checked via web console). I have added a note for myself to keep an eye on this library. 
If an OSGi bundle is available, I will add it as you suggested :).
[1] - https://sling.apache.org/documentation/bundles/sling-query.html

Reply:
ahh... yes, if it's not an OSGI bundle then this is the right approach. Thanks for looking into it!